src/main/js/store/actions/roles.js

```diff
@@ -30,7 +30,7 @@ function fetchedAllRoles(roles) {
 function fetchingAllRolesError(message) {
     return {
         type: USER_MANAGEMENT_ROLE_FETCH_ERROR_ALL,
-        roleFetchError: message
+        message
```

Comment:
Roles table was not showing forbidden permission message.

Reply:
How does this work with no LHS property?

Reply:
Also, this is the roles table.

Reply:
Javascript expands the message variable to be` 'message': { message } ` it's shorthand if the LHS matches the variable name.

Reply:
The roles and users table was not displaying the permission denied banner correctly.  It was not setting the correct attributes to be shown in the UI.  That's why the change was made.